prestodb/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
we have 3 of these now, $root/gradle, $root/prestodb/gradle, and $root/trino/gradle

can we have just 1?